assets/js/index.js

```diff
@@ -108,4 +120,21 @@ function updateFilterCounts(tasks) {
 
 function showNetworkError() {
   backendErrorMessage.innerText = "Something went wrong!";
-}
\ No newline at end of file
+}
+
+async function renderTasks() {
+  const tasks = await getTasks();
+  showTasks(tasks);
+  updateFilterCounts(tasks);
```

Comment:
If you look closely into the code this part of code is repeated in two other places. Maybe you could use this function instead?

Reply:
Done.

Reply:
It's not done yet!